templates/openapi-github/grafbase/schema.graphql

```diff
@@ -0,0 +1,9 @@
+extend schema
+  @openapi(
+    name: "GitHub"
+    url: "https://api.github.com/"
+    schema: "https://raw.githubusercontent.com/github/rest-api-description/main/descriptions/ghes-3.0/ghes-3.0.json"
+    headers: [
+      { name: "Authorization", value: "Bearer {{ env.GITHUB_API_KEY }}" }
+    ]
```

Comment:
Out of interest: have you tried this out?  I didn't even think to try github in my testing

Reply:
It throws the following error (was going to report):

```bash
Couldn't parse HTTP method: patch
Encountered an allOf schema, which we don't currently support
Couldn't parse HTTP method: patch
```

Reply:
😱